path-traversal/extract_files.py

```
@@ -0,0 +1,83 @@
+            file_id = f"{repo}:{file_path}"
+            
+            if file_id not in unique_files:
+                unique_files.add(file_id)
+                
+                if download_file(repo, commit, file_path, output_dir):
+                    downloaded_count += 1
+                    
+                if downloaded_count >= 50:  # Limit to 50 unique files
+                    break
+    
+    print(f"\nSummary:")
```

Ruff (0.13.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)